addons/html_editor/static/src/main/table/table_resize_plugin.js

```diff
@@ -308,28 +309,24 @@ export class TableResizePlugin extends Plugin {
                 target2 = closestElement(ev.target, "tr");
             } else if (isHoveringTdBorder === "right") {
                 if (isRTL) {
-                    target1 = getAdjacentPreviousSiblings(ev.target).find(
-                        (node) => node.nodeName === "TD"
+                    target1 = getAdjacentPreviousSiblings(ev.target).find((node) =>
+                        isTableCell(node)
                     );
```

Comment:
```suggestion
                    target1 = getAdjacentPreviousSiblings(ev.target).find(isTableCell);
```
Same for the other three occurrences below.